dobson/logusers.py

```diff
@@ -0,0 +1,16 @@
+import datetime
+
+import dobson.utils as utils
+
+
+def log_mac_addresses():
+    """Logs all mac addresses connected to the router into a file"""
+    mac_address = [mac.lower() for mac in utils.get_mac_addresses()]
+    log_str = ' '.join(mac_address)
+    this_minute = datetime.datetime.now().replace(second=0, microsecond=0).isoformat()
+    with open(utils.MAC_LOG_FILE, 'a') as f:
+        f.write(f"{this_minute} {log_str}\n")
```

Comment:
Unfortunately the version of Python 3 on the raspberry pi is 3.5.3 (on raspbian stretch), so we can't use F-strings yet :'(

At least format strings are reasonably nice though and still can be used.

Reply:
Yea this was a bit overkill anyway haha, I'll update it.